src/agents/graph.py

```diff
@@ -129,6 +133,7 @@ def _plan(self, state: AgentState) -> dict[str, Any]:
                     )
             except OutputParserException as ope:
                 logger.debug(f"Problem in parsing the planner response: {ope}")
+                # sometimes the plan_response is without 'response' attribute
```

Comment:
It is an interesting fact that you describe here but can you please write a comment that correlates to what is happening in the code like `In case this or that happens we will do this and that.`> In this case maybe something like 'If `plan_response` is missing we do ...'